.github/workflows/auto-cherry-pick.yml

```diff
@@ -62,5 +62,23 @@ jobs:
           reviewers: |
             jerryshao
 
+  cherry_pick_branch_0_8:
+    runs-on: ubuntu-latest
+    name: Cherry pick into branch_0.8
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'branch-0.8') && github.event.pull_request.merged == true }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Cherry pick into branch-0.8
+        uses: carloscastrojumo/github-cherry-pick-action@v1.0.9
+        with:
+          branch: branch-0.8
+          labels: |
+            cherry-pick
+          reviewers: |
```

Comment:
Should the release manager be the reviewer, too?

Reply:
@jerryshao WDYT?

Reply:
I'm OK to add.

Reply:
updated